data.py

```diff
@@ -1,13 +1,9 @@
 from twilio.rest import Client
+from decouple import config
 
-account_sid = 'AC4813fd548f065ee195b1aa2feb96c58f' # Found on Twilio Console Dashboard
-auth_token = 'cda8141ecc84ab6ae0489aacab4d3586' # Found on Twilio Console Dashboard
-myPhone = '+919013908350' # Phone number you used to verify your Twilio account
-TwilioNumber = '+12692206694' # Phone number given to you by Twilio
-
-client = Client(account_sid, auth_token)
+client = Client(config('ACCOUNT_SID'), config('AUTH_TOKEN'))
```

Comment:
Prepend TWILIO in environment variables. `TWILIO_AUTH_TOKEN` vs `AUTH_TOKEN`

Reply:
Done